doc/.vale.ini

```
@@ -1,31 +1,34 @@
-# Core settings
-# =============
-
-# Location of our `styles`
-StylesPath = "styles"
-
-# The options are `suggestion`, `warning`, or `error` (defaults to “warning”).
-MinAlertLevel = warning
-
-# By default, `code` and `tt` are ignored.
-IgnoredScopes = code, tt
-
-# By default, `script`, `style`, `pre`, and `figure` are ignored.
-SkippedScopes = script, style, pre, figure
-
-# WordTemplate specifies what Vale will consider to be an individual word.
-WordTemplate = \b(?:%s)\b
-
-# List of Packages to be used for our guidelines
-Packages = Google
-
-# Define the Ansys vocabulary
-Vocab = ANSYS
-
-[*.{md,rst}]
-
-# Apply the following styles
-BasedOnStyles = Vale, Google
-
-# Removing Google-specific rule - Not applicable under some circumstances
-Google.Colons = NO
+# Core settings
+# =============
+
+# Location of our `styles`
+StylesPath = "styles"
+
+# The options are `suggestion`, `warning`, or `error` (defaults to “warning”).
+MinAlertLevel = warning
+
+# By default, `code` and `tt` are ignored.
+IgnoredScopes = code, tt
+
+# By default, `script`, `style`, `pre`, and `figure` are ignored.
+SkippedScopes = script, style, pre, figure
+
+# WordTemplate specifies what Vale will consider to be an individual word.
+WordTemplate = \b(?:%s)\b
+
+# List of Packages to be used for our guidelines
+Packages = Google
+
+# Define the Ansys vocabulary
+Vocab = ANSYS
+
+[*.{md,rst}]
+
+# Apply the following styles
+BasedOnStyles = Vale, Google
+
+# `class` and `func` are ignored
+TokenIgnores = (:class:`.*`|:func:`.*`|:attr:`.*`|:meth:`.*`)
```

```suggestion
TokenIgnores = (:*:`*.*`)
```
@clatapie can you please try this?

---

Sure, I will! Thank you @Revathyvenugopal162